### DISCUSSION TITLE
Will passthrough cause tests to make real network requests?

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [mswjs/msw#2171] [Q&A] [answered]
This may be a bit of a dumb question. but will using [passthrough](https://mswjs.io/docs/api/passthrough) cause tests to make real network requests?

**Comment by kettanaito** [comment, marked as answer]
Hi, @cmacdonnacha. That's a fantastic question.

Yes, it will. `passthrough()` means "please make this request as-is right now".

**Reply by cmacdonnacha** [reply]
Thanks!
